apps/chrome-devtools/package.json

```diff
@@ -50,10 +46,8 @@
     "@types/chrome": "^0.0.271",
     "@types/jest": "~29.5.2",
     "@types/tinycolor2": "^1.4.6",
-    "@typescript-eslint/eslint-plugin": "^8.0.0",
-    "@typescript-eslint/parser": "^8.0.0",
-    "@typescript-eslint/types": "^8.0.0",
-    "@typescript-eslint/utils": "^8.0.0",
+    "@typescript-eslint/parser": "~8.12.2",
```

Comment:
Already present in `typescript-eslint` but some ESLint plugin didn't migrate yet to this package